app/Healthcheck.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace App;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Config;
+use ORM;
+
+class Healthcheck {
+  const ERROR = 'ERROR';
+  const OK = 'OK';
+  const REDIS_SUCCESSFUL_PING = 'PONG';
+
+  public function index(ServerRequestInterface $request, ResponseInterface $response) {
+    $userlog = make_logger('healthcheck');
+
+    $services = [
+      'mysql' => self::ERROR,
+      'redis' => self::ERROR,
+    ];
+
+    // Redis
+    try {
+      if(redis()->ping() == self::REDIS_SUCCESSFUL_PING) {
+        $services['redis'] = self::OK;
+      }
+    } catch(\Predis\Connection\ConnectionException $e) {
+      $userlog->info(
+        'Redis connection healthcheck failure: ',
+        [
+          'redis_url' => get_redis_url(),
+        ]
+      );
+    }
+
+    // MySQL
+    try {
+      ORM::raw_execute('SELECT version();');
+      $services['mysql'] = self::OK;
+    } catch(\Exception $e) {
+      $userlog->info(
+        'MySQL connection healthcheck failure: ',
+        [
+          'Config::db' => json_encode(Config::$db),
```

Comment:
We should not return the DB config, since this has the potential to also error out in production if there is a temporary network failure for example, and we don't want to expose the database configuration. Can you revise this to just leave out the DB and Redis details?

Reply:
Sure thing. So how would you know what the credentials were (it's a logger, not an echo)

Reply:
You're right I missed that it was logging not outputting to the screen. Still it's safer to not log this. People will have to figure it out.